cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/scheduler/SchedulerQueueResolver.java

```diff
@@ -32,12 +32,12 @@
  */
 public class SchedulerQueueResolver {
   private final String defaultQueue;
-  private final Store store;
+  private final NamespaceStore store;
```

Comment:
I don't think it's useful to call it nsStore, as it's not possible to confuse this "store" with another "store"